legacy/charts/vela-core-legacy/crds/core.oam.dev_traitdefinitions.yaml

```diff
@@ -7,13 +7,6 @@ metadata:
     controller-gen.kubebuilder.io/version: v0.2.4
   name: traitdefinitions.core.oam.dev
 spec:
-  additionalPrinterColumns:
-  - JSONPath: .spec.appliesToWorkloads
-    name: APPLIES-TO
-    type: string
-  - JSONPath: .metadata.annotations.definition\.oam\.dev/description
-    name: DESCRIPTION
-    type: string
```

Comment:
why deleted

Reply:
sorry, it seem some wrong when generate crd file, i will fix it